Services/User/classes/class.ilObjUserGUI.php

```diff
@@ -1195,12 +1195,16 @@ function initForm($a_mode)
 			}
 			else
 			{
-				// begin-patch ldap_multiple
-				#$name = $this->lng->txt('auth_'.$auth_name);
-				include_once './Services/Authentication/classes/class.ilAuthUtils.php';
-				$name = ilAuthUtils::getAuthModeTranslation($auth_key);
-				// end-patch ldap_multiple
-				
+				if ($auth_key >= 100) {
```

Comment:
The plugin slot does currrently not demand auth_keys greater than 99.
The translation of plugin auth modes should be handled in ilAuthUtils::getAuthModeTranslation

Reply:
Yes, you're right - it was a "fast solution" and I think, the plugin auth key need to be higher, then the ILIAS standards, so I thought to set it very high, to not get in conflict, if ILIAS should have new auth per default

I hope so it's now better

Sorry, for the many commits, but PHPStorm doesn't let me remove I unwanted change acording a blank line